src/volunteers.html

```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
 <section>
   <div class="container">
     <h2 class="section-heading">How it works</h2>
-    <div class="col-wrap">
-      <div class="col-3 centered-text">
+    <div class="container">
+      <div class="col-md-4 centered-text" align="center">
```

Comment:
align="center" what does this do? This is not valid inline css (also please no inline css, all css to be in the css files), please can we the bootsrap centering class.